cypress/integration/all-flaky-tests/profile-minister.spec.js

```diff
@@ -320,4 +321,19 @@ context('Testing the application as Minister user', () => {
         .should('not.exist');
     });
   });
+
+  context('Profile rights checks for kort-bestek routes', () => {
+    it.only('check that link to kort bestek zebra view doesnt work', () => {
+      const agendaDate = Cypress.dayjs('2022-04-05');
+
+      cy.openAgendaForDate(agendaDate);
+      cy.get(agenda.agendaActions.optionsDropdown)
+        .children(appuniversum.button)
+        .click();
+      cy.get(agenda.agendaActions.navigateToNewsletter).forceClick();
+      cy.url().should('not.include', 'vergadering/6374F696D9A98BD0A2288559/kort-bestek');
+    });
+
+    // no other tests here because there is no acces to kort bestek
+  });
 });
```

Comment:
`.only` not needed.
Minister profile can see kort-bestek zebra view but will get errors if not released yet.

Reply:
![image](https://user-images.githubusercontent.com/22245223/224064396-f3ef642a-9216-4824-9a59-b2d6080fc41c.png)

Pretty sure all profiles can go to that page